### DISCUSSION TITLE
access_token每次调用都会去获取新的

### DISCUSSION THREAD

**Discussion by mitu-x** [w7corp/easywechat#2762] [Q&A] [answered]
easychat打印的日志，access_token每次调用都会去获取新的。而ticket每一个都能从缓存获取到，请问是怎么回事

wechat.log日志类似下面:
2023年11月11日  12:34:45   ....INFO  :  ......getticket?access_token=123456... ...{' abcdefg '}
2023年11月11日  12:34:46   ....INFO  :  ......getticket?access_token=123678... ...{' abcdefg '}
2023年11月11日  12:34:47   ....INFO  :  ......getticket?access_token=12346436... ...{' abcdefg '}
......
2023年11月11日  14:34:45   ....INFO  :  ......getticket?access_token=1234fsf256... ...{' fsd gagha'}
2023年11月11日  14:34:45   ....INFO  :  ......getticket?access_token=1241241512... ...{' fsd gagha'}
2023年11月11日  14:34:45   ....INFO  :  ......getticket?access_token=123441241... ...{' fsd gagha'}

我记得正常的应该是：
2023年11月11日  12:34:45   ....INFO  :  ......getticket?access_token=123456... ...{' abcdedfdafg '}
2023年11月11日  14:34:46   ....INFO  :  ......getticket?access_token=12342141678... ...{'aff abcfafdefg '}
2023年11月11日  16:34:47   ....INFO  :  ......getticket?access_token=12346hreh436... ...{' afdsfgsbcdefg '}

以上。

希望得到大佬们的解答。

。。。世界和平。。。

**Comment by overtrue** [comment]
缓存怎么配置的？代码在一台机器上还是分布式？

**Reply by mitu-x** [reply, marked as answer]
我是接手的别人的东西，俺也不知道，只是用到大佬easywechat里取ticket和token的功能。俺也不会PHP，就Java重写了个。。。还是感谢大佬的回复，费心了
